lib/linguist/languages.yml

```diff
@@ -625,6 +625,7 @@ CartoCSS:
   language_id: 53
 Ceylon:
   type: programming
+  color: "#563a08"
```

Comment:
I tried a handful of similar colors, but they kept getting rejected. I thought only really dark colors would be left.

Thanks for the hint.